velox/functions/sparksql/Register.cpp

```diff
@@ -459,6 +459,9 @@ void registerFunctions(const std::string& prefix) {
       Array<Generic<T1>>,
       Array<Array<Generic<T1>>>>({prefix + "flatten"});
 
+  registerFunction<RepeatFunction, Varchar, Varchar, int32_t>(
+      {prefix + "string_repeat"});
```

Comment:
I think we can also add the name "repeat" as function alias, as it is Spark's function name. Temporarily we call "string_repeat" in Gluten to avoid conflict issue with presto.